slingpy/evaluation/metrics/auc.py

```diff
@@ -25,11 +25,20 @@ class AreaUnderTheCurve(AbstractMetric):
     """
     Mean absolute error metric.
     """
+
     def __init__(self, multi_class: AnyStr = "ovr"):
         self.multi_class = multi_class
 
     def get_abbreviation(self) -> AnyStr:
         return f"AUC_{self.multi_class}"
 
-    def evaluate(self, y_pred: np.ndarray, y_true: np.ndarray, threshold: Optional[float] = None) -> np.ndarray:
-        return np.asarray(roc_auc_score(y_true[:, 0], y_pred, multi_class="ovr", average="weighted"))
+
+    def evaluate(
+        self, y_pred: np.ndarray, y_true: np.ndarray, threshold: Optional[float] = None
+    ) -> np.ndarray:
+        if y_pred.shape[1] == 2:
```

Comment:
I am conflicted about this solution: this assumes the input is coming from an `SklearnModel` - what if the input is coming from another type of model with 2 outputs? This may mask errors where the model and expected format for regression metrics do not match.

A cleaner solution would be to change [SklearnModel](https://github.com/slingpy/slingpy/blob/master/slingpy/models/sklearn_model.py) to post-process 2D outputs when using a Classification estimator to output values of shape `(n, 1)` in `get_model_prediction`.